src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBatchMultiSource.java

```diff
@@ -83,12 +83,15 @@ public void prepareRun(BatchSourceContext context) throws ConnectionException {
     String sObjectNameField = config.getSObjectNameField();
     context.setInput(Input.of(config.referenceName, new SalesforceInputFormatProvider(
       config, queries, getSchemaWithNameField(sObjectNameField, schemas), sObjectNameField)));
-
+    /* TODO PLUGIN-510
+     *  As part of [CDAP-16290], recordLineage function was introduced with out implementation.
+     *  To avoid compilation errors the code block is commented for future fix.
     Schema schema = context.getInputSchema();
```

Comment:
Why did you need to do this? Your changes look unrelated to lineage

Reply:
Probably you did not follow the google chat on this topic.  To summarize , The current DEVELOP branch does not compile. There was a change introduced last time that caused this. So @CuriousVini  asked to comment the code and add a  TODO.

Reply:
I see, thanks